### DISCUSSION TITLE
原神5.0前瞻兑换码

### DISCUSSION THREAD

**Discussion by yingguan-Cloud** [share121/inter-knot#2388] [其他]
XX3PD82ZYDGN
5WJNV92GGUjW
9FJTU83HYDJA

兑换码将于2024/8/17 12:00过期，请各位旅行者兼绳匠尽快兑换！！！<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by 13346254991** [comment]
在绝区零的绳网上用崩铁表情包发原神兑换码，你是懂的

**Comment by FANG-12-pixel** [comment]
666直播间在逃人员

**Comment by misaka-lastorder** [comment]
难绷

**Comment by YujioNako** [comment]
好人一胎108个！！

**Comment by Kioeco** [comment]
祝你一次108胎！！！！！
